prime-router/src/main/kotlin/azure/observability/event/CodeSummary.kt

```diff
@@ -14,10 +14,10 @@ data class CodeSummary(
         /**
          * Create an instance of [CodeSummary] from a [Coding]
          */
-        fun fromCoding(coding: Coding) = CodeSummary(
-            coding.system ?: UNKNOWN,
-            coding.code ?: UNKNOWN,
-            coding.display ?: UNKNOWN,
+        fun fromCoding(coding: Coding?) = CodeSummary(
```

Comment:
Whats the benefit of calling `fromCoding(null)`? The null check should happen prior to this call.